scripts/baby/baby.cs

```diff
@@ -3,37 +3,40 @@
 
 public partial class Baby : RigidBody2D
 {
-	// Called when the node enters the scene tree for the first time.
-	public override void _Ready()
-	{
-		BodyShapeEntered += OnBodyShapeEntered;
-		BodyEntered += OnBodyEntered;
-	}
+	private float maxHeight;
+	private float maxDistance;
 
-	private void OnBodyEntered(Node body)
-	{
-		GD.Print("I Body Entered", body.Name);
-	}
+	public float MaxHeight { get => maxHeight; set => maxHeight = value; }
+	public float MaxDistance { get => maxDistance; set => maxDistance = value; }
```

Comment:
FYI if you're not adding additional logic on your getters/setters you can just do something like:

```suggestion
	public float MaxHeight { get; set; }
	public float MaxDistance { get; set; }
```

Reply:
Ah yeah fair